src/WebJobs.Script.WebHost/Models/HttpOperationInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.ObjectModel;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    [JsonObject]
+    public class HttpOperationInfo
```

Comment:
Since this is specific to swagger, can we have a more descriptive type name (e.g. `SwaggerHttpOperationInfo`)? Alternatively, we can place these types in a `Swagger` folder under `Models` (with the appropriate namespace change.